trace/tracestate.go

```diff
@@ -121,7 +120,7 @@ func ParseTraceState(tracestate string) (TraceState, error) {
 		}
 
 		if _, ok := found[m.Key]; ok {
-			return TraceState{}, wrapErr(errDuplicate)
+			continue // Duplicate is ignored per https://github.com/w3c/trace-context/blob/551a5b0855171281e98b4c2a814bf9e1f837cd53/test/test.py#L563-L568.
```

Comment:
This is a link to W3C unit tests. Can you instead link to the part of the W3C specification that describes this behavior?

Reply:
I have not found anything better than the test code in the specification that describes it. See: https://github.com/instana/trace-context/blob/master/spec/20-http_request_header_format.md

Reply:
Do you have any other suggestion or can I resolve the comment?

Reply:
It seems like this behavior is only mentioned in the editor draft of the W3C trace-context as optional. This PR seems premature given that unreleased state.

Reply:
Changing to draft

Reply:
> It seems like this behavior is only mentioned in the editor draft of the W3C trace-context as optional. 

Can you paste a quote? I am not sure what fragment are you referring to.

I think that the behavior is not even described and I created https://github.com/w3c/trace-context/issues/548.

Reply:
I think I have found the fragment in the released version of the specification PTAL: https://www.w3.org/TR/trace-context/#combined-header-value

Reply:
> It seems like this behavior is only mentioned in the editor draft of the W3C trace-context as optional. 

For me even the editor draft does not clearly defines this behavior thus I created https://github.com/w3c/trace-context/issues/548